command/thumbnails.php

```diff
@@ -0,0 +1,473 @@
+	 */
+	public function __construct(
+		IUserManager $userManager,
+		IRootFolder $rootFolder,
+		IMountManager $mountManager,
+		ConfigService $configService
+	) {
+		$this->userManager = $userManager;
+		$this->rootFolder = $rootFolder;
+		// IMountManager is 8.2+ only
+		$this->mountManager = $mountManager;
+		$this->configService = $configService;
+		// We want all the media types supported via providers only
+		$this->supportedMediaTypes = $this->configService->getSupportedMediaTypes(true, false);
+
+		parent::__construct();
+	}
+
+	/**
+	 * Processes PHP errors as exceptions in order to be able to keep track of problems
+	 *
+	 * @see https://secure.php.net/manual/en/function.set-error-handler.php
+	 *
+	 * @param int $severity the level of the error raised
+	 * @param string $message
+	 * @param string $file the filename that the error was raised in
+	 * @param int $line the line number the error was raised
+	 *
+	 * @throws \ErrorException
+	 */
+	public function exceptionErrorHandler($severity, $message, $file, $line) {
+		if (!(error_reporting() & $severity)) {
+			// This error code is not included in error_reporting
+			return;
+		}
+		throw new \ErrorException($message, 0, $severity, $file, $line);
+	}
+
+	/**
+	 * Initialises some useful tools for the Command
+	 */
+	protected function initTools() {
+		// Start the timer
+		$this->execTime = -microtime(true);
+
+		// Convert PHP errors to exceptions
+		set_error_handler([$this, 'exceptionErrorHandler'], E_ALL);
+
+		// Collect interrupts and notify the running command
+		pcntl_signal(SIGTERM, [$this, 'cancelOperation']);
+		pcntl_signal(SIGINT, [$this, 'cancelOperation']);
+	}
+
+	/**
+	 * Parses all the parameters submitted by the command user
+	 *
+	 * @param InputInterface $input
+	 * @param string $userInputErrorMsg
+	 * @param string $pathInputErrorMsg
+	 * @param OutputInterface $output
+	 *
+	 * @return array
+	 * @throws InputCommandException
+	 */
+	protected function parseParameters(
+		InputInterface $input, $userInputErrorMsg, $pathInputErrorMsg, OutputInterface $output
+	) {
+		$inputPath = $input->getOption('path');
+		if ($inputPath) {
+			$inputPath = '/' . trim($inputPath, '/');
+			try {
+				list (, $user, $topFolder) = explode('/', $inputPath, 4);
+				$users = [$user];
+			} catch (\ErrorException $exception) {
+				throw new InputCommandException($pathInputErrorMsg);
+			}
+			$this->checkPathParameter($topFolder, $pathInputErrorMsg);
+		} else if ($input->getOption('all')) {
+			$users = $this->userManager->search('');
+		} else {
+			$users = $input->getArgument('user_id');
+		}
+		$this->checkUserParameter($users, $userInputErrorMsg);
+
+		return [$inputPath, $users];
+	}
+
+	/**
+	 * Presents the results of the operation in tabular form
+	 *
+	 * @param OutputInterface $output
+	 */
+	protected function presentResults(OutputInterface $output) {
+		// Stop the timer
+		$this->execTime += microtime(true);
+
+		$output->setVerbosity($output::VERBOSITY_NORMAL);
```

Comment:
if you change the `$output::VERBOSITY_*` stuff to `OutputInterface::VERBOSITY_*` it solves the issue with the check-code